javaagent/build.gradle.kts

```diff
@@ -218,6 +218,25 @@ tasks {
   named("generateLicenseReport").configure {
     dependsOn(cleanLicenses)
   }
+
+  // Because we reconfigure publishing to only include the shadow jar, the Gradle metadata is not correct.
```

Comment:
This was actually already the case, causing awkwardness like this

https://github.com/open-telemetry/opentelemetry-lambda/blob/main/java/layer-javaagent/build.gradle.kts#L12